Api/csharp/ArmoniK.Api.Client/Submitter/SubmitterClientExt.cs

```diff
@@ -225,39 +180,47 @@ await stream.RequestStream.CompleteAsync()
                                      DataComplete = true,
                                    },
                    };
-
-      if (isLast)
-      {
-        yield return new CreateLargeTaskRequest
-                     {
-                       InitTask = new InitTaskRequest
-                                  {
-                                    LastTask = true,
-                                  },
-                     };
-      }
     }
 
     /// <summary>
-    ///   Get result without streaming
+    ///   Get result as a stream
     /// </summary>
     /// <param name="client">gRPC client to the Submitter</param>
     /// <param name="resultRequest">Request for result</param>
     /// <param name="cancellationToken">Token used to cancel the execution of the method</param>
     /// <returns>
-    ///   A byte array containing the data associated to the result
+    ///   A stream in which the data will be written
+    /// </returns>
+    /// <exception cref="Exception">a result reply chunk is not data, rending it impossible to reconstitute the data</exception>
+    /// <exception cref="ArgumentOutOfRangeException">result reply type is unknown</exception>
+    [PublicAPI]
+    public static Task<Stream> GetResultAsStreamAsync(this gRPC.V1.Submitter.Submitter.SubmitterClient client,
```

Comment:
@aneojgurhem With the stream correctly implemented, it is not necessary for this function to be async. Do we remove the async?

Reply:
We can keep it for now